AK/NonnullPtrVector.h

```diff
@@ -25,7 +25,7 @@ class NonnullPtrVector : public Vector<PtrType, inline_capacity> {
     {
     }
     NonnullPtrVector(const Vector<PtrType>& other)
```

Comment:
Leaving aside the sheer size of this change (which would probably be better off as multiple commits), it doesn't seem to be catching all the `const T`'s.